packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -357,9 +385,12 @@ export abstract class QueueProcessingServiceBase extends Construct {
    */
   protected configureAutoscalingForService(service: BaseService) {
     const scalingTarget = service.autoScaleTaskCount({ maxCapacity: this.maxCapacity, minCapacity: this.minCapacity });
-    scalingTarget.scaleOnCpuUtilization('CpuScaling', {
-      targetUtilizationPercent: 50,
-    });
+
+    if (!this.disableCpuBasedScaling) {
+      scalingTarget.scaleOnCpuUtilization('CpuScaling', {
+        targetUtilizationPercent: this.cpuTargetUtilizationPercent ?? 50,
```

Comment:
```suggestion
        targetUtilizationPercent: this.cpuTargetUtilizationPercent,
```
The default value is already assigned above.

Reply:
Thanks, updated.